perun-base/src/test/resources/perun-roles.yml

```diff
@@ -24,8 +24,108 @@ perun_roles:
 #A list of Perun policies that are loaded to the PerunPoliciesContainer.
 perun_policies:
 
+  test_resourceselfservice:
+    policy_roles:
+       - RESOURCESELFSERVICE: Resource
```

Comment:
RESOURCESELFSERVICE role will be always in the rule together with the GROUPADMIN role (AND connection between them). Please add the "GROUPADMIN: Group" to the rule and correct the tests if needed.